.github/workflows/ci.yml

```diff
@@ -246,7 +246,7 @@ jobs:
       matrix:
         node: [14, 16]
         cmd:
-          - CLIENT=selenium:firefox npm run test-webpack
+          - git reset --hard && CLIENT=selenium:firefox npm run test-webpack
```

Comment:
Adding this line has actually changed the name of the GH Action which means the expected action in the branch protection rules is no longer running. I could change that but it's now an ugly name. Could you add the `git reset --hard` to the `steps` instead of the `cmd`?